Release 1.58.2 does not include the `vendor` folder containing `embedded-protocol`

After the 1.58.2 we started to get the following CSS build errors at https://github.com/decidim/decidim:

```
ERROR in ../../decidim-accountability/app/packs/stylesheets/redesigned_accountability.scss (../../decidim-accountability/app/packs/stylesheets/redesigned_accountability.scss.webpack[javascript/auto]!=!./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[4].use[1]!./node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[4].use[2]!./packages/webpacker/src/loaders/decidim-sass-loader.js!../../decidim-accountability/app/packs/stylesheets/redesigned_accountability.scss)
Module build failed (from ./packages/webpacker/src/loaders/decidim-sass-loader.js):
Error: Cannot find module './vendor/embedded_sass_pb'
Require stack:
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/src/utils.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/src/value/index.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/src/value/boolean.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/index.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/packages/webpacker/src/loaders/decidim-sass-loader.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/loadLoader.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/webpack/lib/NormalModule.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/webpack/lib/index.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/webpack-cli/lib/webpack-cli.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/webpack-cli/lib/bootstrap.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/webpack-cli/bin/cli.js
- /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/webpack/bin/webpack.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/src/utils.js:10:15)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/src/value/index.js:8:17)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/src/value/boolean.js:8:17)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/sass-embedded/dist/lib/index.js:8:19)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/packages/webpacker/src/loaders/decidim-sass-loader.js:1:14)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at loadLoader (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/loadLoader.js:19:17)
    at iteratePitchingLoaders (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js:182:2)
    at iteratePitchingLoaders (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js:178:10)
    at /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js:189:18
    at handleResult (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/loadLoader.js:53:2)
    at loadLoader (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/loadLoader.js:35:10)
    at iteratePitchingLoaders (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js:182:2)
    at iteratePitchingLoaders (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js:178:10)
    at /home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/LoaderRunner.js:189:18
    at handleResult (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/loadLoader.js:53:2)
    at loadLoader (/home/runner/work/decidim/decidim/spec/decidim_dummy_app/node_modules/loader-runner/lib/loadLoader.js:35:10)
 @ ../../decidim-accountability/app/packs/stylesheets/redesigned_accountability.scss
 @ ../../decidim-accountability/app/packs/entrypoints/decidim_accountability.js 4:0-51
```

After further inspection, it seems that the 1.58.2 release does not contain the `dist/lib/src/vendor` folder which includes the missing file it is trying to require.

In the 1.57.1 this folder was included and there are no such build errors when building the CSS with that version.

I'm seeing something similar today with an older release version 1.56.1: `Error: Cannot find module './vendor/embedded_sass_pb'`

Suddenly broken, without changing anything on our end.

--- a/tool/prepare-release.ts
+++ b/tool/prepare-release.ts
@@ -24,11 +24,6 @@ import {getJSApi} from './get-js-api';
     shell.cp('-R', 'lib/src/vendor/sass', 'dist/types');
     await fs.unlink('dist/types/README.md');
 
-    // .gitignore needs to exist in dist for `npm publish` to correctly exclude
-    // files from the published tarball.
-    console.log('Copying .gitignore to dist.');
-    await fs.copyFile('.gitignore', 'dist/.gitignore');
-
     console.log('Ready for publishing to npm.');
   } catch (error) {
     console.error(error);

